instruqt/distributing-with-replicated/04-installing-the-application/setup-shell

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# This set line ensures that all failures will cause the script to error and exit
+set -euxo pipefail
+source /etc/profile.d/header.sh
+
+# get the customer id, since it's the password for the Helm installation
+# and users like to copy/paste
+api_token=$(get_api_token)
+app_id=$(curl --header 'Accept: application/json' --header "Authorization: ${api_token}" https://api.replicated.com/vendor/v3/apps | jq -r '.apps[0].id')
+customer_id=$(curl --header 'Accept: application/json' --header "Authorization: ${api_token}" "https://api.replicated.com/vendor/v3/app/${app_id}/customers" | jq -r '.customers[0].id')
+
+# set the customer email so the user doesn't have to do it, also make it a paid license for the optics
+customer_email="${INSTRUQT_PARTICIPANT_ID}@omozan.io"
+updated_customer=$(curl --header 'Accept: application/json' --header "Authorization: ${api_token}" "https://api.replicated.com/vendor/v3/app/${app_id}/customer/${customer_id}" | \
+  jq -c --arg appId "${app_id}" --arg email "${customer_email}" '.customer | {"app_id": $appId, "name": .name, "email": $email, "channel_id": .channels[0].id, "expires_at": .expiresAt, "type": "prod", "is_airgap_enabled": .airgap, "is_gitops_supported": .isGitopsSupported, "is_identity_service_supported": .isIdentityServiceSupported, "is_geoaxis_supported": .isGeoaxisSupported, "is_snapshot_supported": .isSnapshotSupported, "is_support_bundle_upload_enabled": .isSupportBundleUploadEnabled, "entitlementValues":[]}')
```

Comment:
this is probably where assigning the channel also happens. Long term this should go into the CLI. Or the npm module (which can already do some of this while creating the customer)
https://github.com/replicatedhq/reliability-actions-lib/blob/main/src/customers.ts#L29